indico/modules/events/abstracts/models/abstracts.py

```diff
@@ -32,8 +32,8 @@
 from indico.modules.events.abstracts.models.review_questions import AbstractReviewQuestion
 from indico.modules.events.abstracts.models.review_ratings import AbstractReviewRating
 from indico.modules.events.models.persons import AuthorsSpeakersMixin
+from indico.modules.events.models.reviews import ProposalMixin
 from indico.modules.events.contributions.models.contributions import _get_next_friendly_id, CustomFieldsMixin
```

Comment:
What's wrong with this exactly? We are making use of both `_get_next_friendly_id` and `CustomFieldsMixin`.